### PR TITLE
find, stay simple

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
modules/find

### CODE REVIEW

File: lib/ansible/modules/find.py [hunk 1]
```diff
@@ -19,6 +19,9 @@
 description:
     - Return a list of files based on specific criteria. Multiple criteria are AND'd together.
     - For Windows targets, use the M(ansible.windows.win_find) module instead.
+    - This module does not use the C(find) command, it a much simpler and slower Python implementation.
```
Comment (gotmax23) [inline lib/ansible/modules/find.py]: Comma splice

File: lib/ansible/modules/find.py [hunk 2]
```diff
@@ -19,6 +19,9 @@
...
+      It is inteded for small and simple uses. Those that need the extra power or speed and have expertise
+      with the Unix command, should use it directly.
```
Comment (gotmax23) [inline lib/ansible/modules/find.py]:
```suggestion
      with the Unix command should use it directly.
```

File: lib/ansible/modules/find.py (hunk 1 above)
Comment (Akasurde) [inline lib/ansible/modules/find.py]:
```suggestion
    - This module does not use the C(find) command, it is a much simpler and slower Python implementation.
```

File: lib/ansible/modules/find.py [hunk 3]
```diff
@@ -19,6 +19,9 @@
 description:
     - Return a list of files based on specific criteria. Multiple criteria are AND'd together.
     - For Windows targets, use the M(ansible.windows.win_find) module instead.
+    - This module does not use the C(find) command, it a much simpler and slower Python implementation.
+      It is inteded for small and simple uses. Those that need the extra power or speed and have expertise
```
Comment (Akasurde) [inline lib/ansible/modules/find.py]:
```suggestion
      It is intended for small and simple uses. Those that need the extra power or speed and have expertise
```

File: lib/ansible/modules/find.py (hunk 2 above)
Comment (Akasurde) [inline lib/ansible/modules/find.py]:
```suggestion
      with the UNIX command, should use it directly.
```

**Review by Akasurde — APPROVED** [review]

**Pull request merged**